opennms-doc/guide-install/src/asciidoc/text/opennms/windows.adoc

```diff
@@ -0,0 +1,99 @@
+. Download and install the graphical _OpenNMS_ installer
+. First start of the _OpenNMS_ application
+
+[[gi-install-opennms-deb-prepare-pg]]
+==== Installation PostgreSQL
+
+_PostgreSQL_ is available for _Microsoft Windows_ and latest version can be downloaded from link:http://www.enterprisedb.com/products-services-training/pgdownload#windows[Download PostgreSQL] page.
+Follow the on-screen instructions of the graphical installer.
+
+NOTE: The placeholder `{PG-VERSION}` represents the _PostgreSQL_ version number.
+      A version of _9.1+_ is required for _OpenNMS_.
+
+The following information has to be provided:
+
+* Installation Directory for _PostgreSQL_, e.g. `C:\Program Files\PostgreSQL\{PG-VERSION}`
+* Password for the database superuser (_postgres_), this password will be used during the _OpenNMS_ setup.
+* Port to listen for _PostgreSQL_ connections, default is `5432` and can normally be used.
+* Locale for the database, keep `[Default locale]`, if you change the locale, _OpenNMS_ may not be able to initialize the database.
+
+TIP: It is not required to install anything additional from the _PostgreSQL Stack Builder_.
+
+NOTE: The database data directory is automatically initialized during the setup and the _postgresql-x64-{PG-VERSION}_ is already added as service and automatically started at system boot.
+
+NOTE: It is not necessary to change the authentication method in `pg_hba.conf`, it is by default set to `md5` for localhost connections.
+
+[[gi-install-opennms-deb-repo]]
+==== Install OpenNMS with GUI installer
+
+For _Microsoft Windows_ environments download the _standalone-opennms-installer-{VERSION-NUMBER}.zip_ file from the link:http://sourceforge.net/projects/opennms/files/OpenNMS/[OpenNMS SourceForge] repository.
+Extract the downloaded _ZIP_ file.
+
+NOTE: The `{ONMS-VERSION}` has to be replaced with the latest stable version.
+
+Start the graphical installer and follow the on screen instructions.
+The following information has to be provided:
+
+* Path to _Oracle JDK_, e.g. `C:\Program Files\Java\jdk1.8.0_51`
+* Installation path for _OpenNMS_, e.g. `C:\Program Files\OpenNMS`
+* Select packages which has to be installed, the minimum default selection is _Core_ and _Docs_
+* PostgreSQL Database connection
+** Host: Server with _PostgreSQL_ running, e.g. `localhost`
+** Name: Database name for _OpenNMS_, e.g. `opennms`
+** Port: _TCP_ port connecting to _PostgreSQL_ server, e.g. `5432`
+** Username (administrative superuser): _PostgreSQL_ super user, e.g. `postgres`
+** Password (administrative superuser): Password given during _PostgreSQL_ setup for the super user
+** Username (runtime user for opennms): Username to connect to the _OpenNMS_ database, e.g. `opennms`
+** Password (runtime user for opennms): Password to connect to the _OpenNMS_ database, e.g. `opennms`
+* Configure a discovery range for an initial node discovery, if you don't want discovery set begin and end to the same unreachable address.
```

Comment:
Configure a discovery range for an initial node discovery. If you don't want any discovery set begin and end to the same unreachable address.